packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -458,6 +468,19 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final Radius cursorRadius;
 
+  /// Whether the cursor will animate from fully transparent to fully opaque
+  /// during each cursor blink.
+  ///
+  /// By default, the cursor opacity will animate on iOS platforms and will not
+  /// animate on Android platforms.
+  final bool cursorOpacityAnimates;
```

Comment:
Random thought: could we just have the composer pass in a Curve instead of a bool? This way, Android passes a step function, iOS passes something linear (or some such. I don't really know). But then fuchsia can pass in something else and our users can pass in whatever they like too.

Reply:
The way things are set up right now, doing this would be non trivial... But if there's a need for it in the future, we could definitely add that.

Reply:
Sure. Just try to account letting it be non-API-breaking when leaving room for it.